packages/@lwc/template-compiler/src/__tests__/fixtures/svg/valid-svg/expected.js

```diff
@@ -1,119 +1,8 @@
-import { registerTemplate } from "lwc";
-const stc0 = {
-  key: 0,
-  svg: true,
-};
-const stc1 = {
-  key: 1,
-  svg: true,
-};
-const stc2 = {
-  key: 2,
-  svg: true,
-};
-const stc3 = {
-  key: 3,
-  svg: true,
-};
-const stc4 = {
-  key: 4,
-  svg: true,
-};
-const stc5 = {
-  key: 5,
-  svg: true,
-};
-const stc6 = {
-  key: 6,
-  svg: true,
-};
-const stc7 = {
-  key: 7,
-  svg: true,
-};
-const stc8 = {
-  key: 8,
-  svg: true,
-};
-const stc9 = {
-  key: 9,
-  svg: true,
-};
-const stc10 = {
-  key: 10,
-  svg: true,
-};
-const stc11 = {
-  key: 11,
-  svg: true,
-};
-const stc12 = {
-  key: 12,
-  svg: true,
-};
-const stc13 = {
-  key: 13,
-  svg: true,
-};
-const stc14 = {
-  key: 14,
-  svg: true,
-};
-const stc15 = {
-  key: 15,
-  svg: true,
-};
-const stc16 = {
-  key: 16,
-  svg: true,
-};
-const stc17 = {
-  key: 17,
-  svg: true,
-};
-const stc18 = {
-  key: 18,
-  svg: true,
-};
-const stc19 = {
-  key: 19,
-  svg: true,
-};
-const stc20 = {
-  key: 20,
-  svg: true,
-};
-const stc21 = {
-  key: 21,
-  svg: true,
-};
+import { parseFragment, registerTemplate } from "lwc";
+const $fragment1 = parseFragment`<svg${1}${2}><a${1}${2}></a><circle${1}${2}><!-- &lt;clipPath&gt;&lt;/clipPath&gt; --><defs${1}${2}></defs><desc${1}${2}></desc><ellipse${1}${2}><filter${1}${2}></filter><g${1}${2}></g><line${1}${2}><marker${1}${2}></marker><mask${1}${2}></mask><path${1}${2}><pattern${1}${2}></pattern><polygon${1}${2}></polygon><polyline${1}${2}></polyline><rect${1}${2}><stop${1}${2}></stop><symbol${1}${2}></symbol><text${1}${2}></text><!-- &lt;textPath&gt;&lt;/textPath&gt; --><title${1}${2}></title><tspan${1}${2}></tspan><use${1}${2}></use></svg>`;
```

Comment:
A comment was added in the generated output `<!-- &lt;clipPath&gt;&lt;/clipPath&gt; -->`.
It appears that the static codegen doesn't respect the `lwc:comment` directive.